0-bootstrap/variables.tf

```diff
@@ -84,6 +84,7 @@ variable "skip_gcloud_download" {
 //}
 //
 //variable "jenkins_master_ip_addresses" {
+//  // TODO: rename to jenkins_master_subnetwork_cidr_range
```

Comment:
Need to update the variable name and remove the TODO

Reply:
Done. Thank you!